R/exposure_metrics.R

```diff
@@ -132,3 +132,71 @@ calc_auc_from_regimen <- function(
     target_design$type
   )
 }
+
+#' Calculate time to target attainment
+#'
```

Comment:
I think it would be helpful to provide more documentation for the user about how time to target is defined. For example, how dose index is defined, that the time involves the infusion length, etc.